root/view/src/main/java/at/fhv/itb5c/view/user/UserModel.java

```diff
@@ -0,0 +1,145 @@
+	private DoubleProperty _membershipFee;
+	private ObservableSet<UserRole> _userRoles;
+
+	private IUserRMI _rmiUser;
+	
+	private UserModel(IUserRMI user) throws RemoteException {
+		_firstName = new SimpleStringProperty();
+		_lastName = new SimpleStringProperty();
+		_address = new SimpleStringProperty();
+		_telephonenumber = new SimpleStringProperty();
+		_eMail = new SimpleStringProperty();
+		_birthDate = new SimpleObjectProperty<>();
+		_membershipFee = new SimpleDoubleProperty();
+		
+		_firstName.setValue(user.getFirstName());
+		_lastName.setValue(user.getLastName());
+		_address.setValue(user.getAddress());
+		_eMail.setValue(user.getEmail());
+		_telephonenumber.setValue(user.getTelephoneNumber());
+		_gender = user.getGender();
+		_birthDate.setValue(user.getDateOfBirth());
+		_typeOfSports = (ObservableSet <TypeOfSport>) FXCollections.observableSet(user.getTypeOfSports());
+		_membershipFee.setValue(user.getMembershipFee());	
+		_userRoles = (ObservableSet<UserRole>) FXCollections.observableSet(user.getRoles());
+		
+		_rmiUser = user;
+	}
+	
+	public static UserModel createUserModel(IUserRMI user) throws RemoteException {
+		return new UserModel(user);
+	}
+	
+	public IUserRMI getRMIUser() {
+		try {
+			_rmiUser.setFirstName(_firstName.getValue());
+			_rmiUser.setLastName(_lastName.getValue());
+			_rmiUser.setAddress(_address.getValue());
+			_rmiUser.setTelephoneNumber(_telephonenumber.getValue());
+			_rmiUser.setEmail(_eMail.getValue());
+			_rmiUser.setDateOfBirth(_birthDate.getValue());
+			_rmiUser.setMembershipFee(_membershipFee.getValue());
+			
+		} catch (RemoteException e) {
+			AlertUtil.ConnectionAlert();
+		}
+		
+		return _rmiUser;
+	}
+	
+	public StringProperty getFirstName() {
+		return _firstName;
+	}
+	
+	public StringProperty getLastName() {
+		return _lastName;
+	}
+
+	public StringProperty getAdress() {
+		return _address;
+	}
+
+	public StringProperty getTelephonenumber() {
+		return _telephonenumber;
+	}
+
+	public StringProperty getEMail() {
+		return _eMail;
+	}
+
+	public void setGender(Gender gender) {
+		try {
+			_rmiUser.setGender(gender);
+		} catch (RemoteException e) {
+			AlertUtil.ConnectionAlert();
+		}
+		_gender = gender;
+	}
+	
+	public Gender getGender() {
+		return _gender;
+	}
+	
+	public ObjectProperty<LocalDate> getBirthDate() {
+		return _birthDate;
+	}
+	
+	public ObservableSet<TypeOfSport> getTypeOfSports() {
+		return _typeOfSports;
+	}
+	
+	public void setTypeOfSports(ObservableSet<TypeOfSport> typeOfSports) {
+		try {
+			_rmiUser.setTypeOfSports(new HashSet<>(typeOfSports));
+		} catch (RemoteException e) {
+			AlertUtil.ConnectionAlert();
+		}
+		_typeOfSports = typeOfSports;
+	}
+	
+	public DoubleProperty getMemberShipFee() {
+		return _membershipFee;
+	}
+	
+	public ObservableSet<UserRole> getUserRoles() {
+		return _userRoles;
+	}
+	
+	public void setUserRoles(ObservableSet<UserRole> userRoles) {
+		try {
+			_rmiUser.setRoles(new HashSet<>(userRoles));
```

Comment:
What if userRoles == null? Wouldn't this result in an uncaught NullPointerException?